about.html

```diff
@@ -0,0 +1,176 @@
+   
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Final Capstone</title>
+</head>
+
+<body>
+    <nav class="main-nav mainn-nav nav">
+        <div class="container">
+            <ul class="main-nav-ul float-right">
+                <li class="main-nav"><i class="fa-brands fa-facebook-f"></i></li>
+                <li class="main-nav"><i class="fa-brands fa-twitter"></i></li>
+                <li class="main-nav">English</li>
+                <li class="main-nav">My Page</li>
+                <li class="main-nav">Logout</li>
+            </ul>
+        </div>
+    </nav>
+    <nav class="second-nav">
+
+        <span class="ham">
+            <a href="#" id="ham"><i class="fas fa-bars"></i></a>
+           
+        </span>
+        <span class="hide">
+            <i id="hide" class="fa-solid fa-xmark fa-3x"></i>
+         </span>
+         
+        <div class="navigation">
+            <!-- <img class="navigation-logo" src="img/about_logo 1.png" alt=""> -->
+            <!-- <img class="navigation-logo" src="Img/Logo.png" alt="Logo"> -->
+            <!-- <h3 class="navigation-logos"></h3> -->
+            <h4 id = "logos"><a href="index.html">Logo</a></h4>
+            
+            <ul class="second-nav-ul">
+                <li><a class="second-nav-i" href="index.html">Home</a></li><br>
+                <li><a class="second-nav-i" href="#program">Program</a></li><br>
+                <li><a class="second-nav-i" href="#join">Join</a></li><br>
+                <li><a class="second-nav-i" href="#sponsor">Sponsor</a></li><br>
+                <li><a class="second-nav-i" href="#updates">News</a></li><br>
+                <li><a href="#" class="mt-3 p-2" id="cc-btn">Campaign</a></li><br>
+            </ul>
+        </div>
+
+    </nav>
+
+    <nav class="mob-nav">
+        <Ul class="mobiles-menu">
+           <li><a id="active mobile-class" href="index.html">Home</a></li>
+           <hr id="horizontal" />
+           <li><a id="mobile-class" href="#about">Program</a></li>
+           <hr id="horizontal" />
+           <li><a id="mobile-class" href="#contact">Join</a></li>
+           <hr id="horizontal"/>
+           <li><a id="mobile-class" href="#contact">Sponsor</a></li>
+           <hr id="horizontal" />
+           <li><a id="mobile-class" href="#contact">News</a></li>
+           <hr id="horizontal" />
+        </Ul>
+     </nav>
+    <section class="about main-section">
+
+        <div class="main-text main-abt">
+            <!-- <h6 class ="top">"Hello! Tech world"</h6> -->
+            <h2 class="middle">African Technology Congress
+                2024!</h2>
```

Comment:
- [ ] I suggest that you consider using `h1` Html tag instead of `h2` tag. The H1 tag allows Google to know the title of a page. Using this to then define the main subject of the page and its general theme which then improves the SEO of the application ☺️. You should always ensure that each page has one `h1` HTML tag.

Reply:
- [ ] @Zel-hub7 as previously requested kindly change the `h2` tag  to `h1` tag , this will make your page suitable for improving SEO such as Google while  ranking the page